cornell-dti/cu-apts
# Connect review to apartment cards

### Summary 

This PR is for changing the "Reviews" button in the navbar to now connect to a new reviews page that is populated with cards where each card represents one of the buildings in the db. Each card links to the landlord reviews for the renting company that owns that particular building. 

This PR also includes a lot of code refactoring: the apartment cards are now their own component, and the previous `/homepageData` endpoint was changed to a generic `/page-data/:page` endpoint that supports retrieving data for the three apartment cards on the home page as well as the apartment cards on the reviews page. If `page` is `'home'`, then data for only three buildings is returned. Otherwise, data for all buildings in the db is returned. 



https://user-images.githubusercontent.com/57203639/118387766-476b7280-b5ee-11eb-931e-6a88bb3eb8f5.mov



### Test Plan 

Run `yarn start` in the root directory to start a local dev server. 
- Click on the "Reviews" button in the navbar to see the reviews page. 
- Click on an apartment card to navigate to the landlord reviews, and click on the "Reviews" button in the navbar to see the reviews page. 

### Notes
This PR is based off of #70 because #70 fixes a major bug in one of the endpoints.
